src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Vaccination/HomeVaccinationCellModel.swift

```diff
@@ -0,0 +1,46 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class HomeVaccinationCellModel {
+
+	// MARK: - Init
+
+	init(
+		healthCertifiedPerson: HealthCertifiedPerson,
+		onUpdate: @escaping () -> Void
+	) {
+		self.healthCertifiedPerson = healthCertifiedPerson
+		healthCertifiedPerson.$hasValidProofCertificate.sink { isValid in
+			self.isVerified = isValid
+			onUpdate()
+		}.store(in: &subscriptions)
+		DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
+			self.isVerified = true
+			onUpdate()
+		}
+		healthCertifiedPerson.$proofCertificate.sink(receiveValue: { proofCertificate in
+			guard let proofCertificate = proofCertificate else {
+				self.vaccinatedPersonName = healthCertifiedPerson.healthCertificates.first?.name.fullName
```

Comment:
Would be better to check for existence before assigning.

Reply:
we have to use the value  if there was no proof certificate available